### DISCUSSION TITLE
Total GHG emissions by NAICS-6

### DISCUSSION THREAD

**Discussion by nacho-osio** [USEPA/supply-chain-factors#59] [Q&A] [answered]
Hi,

As versions v1.1.1 and v1.2 of GHG emission factors are classified by NACIS-6, I would like to access the total emissions by NAICS-6 as well. 

Where could I get this information?

**Comment by bl-young** [comment, marked as answer]
The emission factors are based on totals developed in EPA’s [FLOWSA](https://github.com/USEPA/flowsa) python package. You can see we flag the specific dataset in the USEEIO model spec file [here](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml#L19). That file can be downloaded from EPA’s Data Commons:
https://dmap-data-commons-ord.s3.amazonaws.com/flowsa/FlowBySector/GHG_national_2019_m1_v1.3.1_d680131.parquet

These are in a `.parquet` data type format which are essentially compressed csv files. These can be read in python or R, for example, if you have the useeior package: 
```{R}
f <- 'flowsa/FlowBySector/GHG_national_2019_m1_v1.3.1_d680131.parquet'
file <- useeior:::loadDataCommonsfile(f)
df <- as.data.frame(arrow::read_parquet(file))
```
